media/js/firefox/new/scene1.js

```diff
@@ -120,4 +120,19 @@
         initOtherPlatformsModal();
     }
 
+    /**
+     * If user is on iOS and has followed an adjust.com link, redirect to
+     * the App Store.
+     * https://github.com/mozilla/bedrock/issues/5629
+     */
+    // TODO: is this a solid enough check?
+    if (window.site.platform === 'ios' && /open\-app\-store=true/.test(window.location.search)) {
+        var appStoreURL = $('.os_ios a').attr('href');
```

Comment:
Maybe make this selector a little more specific just for good measure? `.download-button .ios_ios > .download-link`